tailwind/plugins/base/gc-base.js

```diff
@@ -11,6 +11,7 @@ module.exports = function () {
       },
       body: {
         fontSize: '1.8rem',
+        color: '#17171A',
```

Comment:
This makes sure that the fallback color for text everywhere is this Campaign Black and not the softer Decanter black.